File: src/Dotnet9.WebAPI/Dotnet9.Application/Client/Dtos/ArticleListQueryInput.cs
namespace Dotnet9.Application.Client.Dtos;

public class ArticleListQueryInput : Pagination
{
    /// <summary>
    /// 标签名
    /// </summary>
    public string? TagName { get; set; }

    /// <summary>
    /// 分类别名
    /// </summary>
    public string? CategorySlug { get; set; }

    /// <summary>
    /// 专辑ID
    /// </summary>
    public string? AlbumSlug { get; set; }

    /// <summary>
    /// 关键词
    /// </summary>
    public string Keyword { get; set; }
}

File: src/Dotnet9.VueBlogSSR/api/models/article-list-query-input.ts
import { Pagination } from "./pagination";

export interface ArticleListQueryInput extends Pagination {
  /**
   * 标签ID
   */
  tagId?: number;

  /**
   * 分类ID
   */
  categoryId?: number;
  /**
   * 关键词
   */
  keyword?: string;
}
